partner_identification/models/res_partner.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2004-2010 Tiny SPRL http://tiny.be
+# Copyright 2010-2012 ChriCar Beteiligungs- und Beratungs- GmbH
+#             http://www.camptocamp.at
+# Copyright 2015 Antiun Ingenieria, SL (Madrid, Spain)
+#        http://www.antiun.com
+#        Antonio Espinosa <antonioea@antiun.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, fields, _
+from odoo.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    id_numbers = fields.One2many(
+        comodel_name='res.partner.id_number',
+        inverse_name='partner_id',
+        string="Identification Numbers",
+    )
+
+    @api.multi
+    @api.depends('id_numbers')
+    def _compute_identification(self, field_name, category_code):
```

Comment:
Does it really make sense to put ``api.depends``? Will it work when we use it via lambda?
```

 compute=lambda s: s._compute_identification(
                    'social_security', 'SSN',
                ),
```